Corzbank.Data/Entities/Transfer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Corzbank.Data.Enums;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Corzbank.Data.Entities
+{
+    public class Transfer: KeyEntity
+    {
+        [Required(ErrorMessage = "TransferType is required")]
```

Comment:
Required is by default. It was mentioned yesterday. You don't need such attributes with error messages for entities. Remove from all the places

Reply:
resolved